services/main.go

```diff
@@ -104,12 +104,13 @@ func (server *ApiService) productsHandler(c *fiber.Ctx) error {
 	}
 
 	if server.Mode == Trial {
+		data = append(data, constants.CHEF_INFRA_CLIENT_ENTERPRISE_PRODUCT)
```

Comment:
please add this in `server.DynamoServices(server.DatabaseService, c).Products(data, params.Eol)` function